explanations/python-explicit-check-phase.md

```diff
@@ -0,0 +1,42 @@
+Nixpkgs contains a `pytestCheckHook` that can automatically run tests using pytest.
+
+For example, rather than
+
+```
+buildPythonPackage {
+    checkInputs = [ pytest ...];
+    checkPhase = ''
+      pytest
+    '';
+};
+```
+
+Consider instead using
+```
+buildPythonPackage {
+    checkInputs = [ pytestCheckHook ...];
+};
+
+```
+
+Also note that many flags that you might want to pass to `pytest` can be passed automatically through `pytestCheckHook`.
+
+`buildPythonPackage` accepts:
+
+ - `disabledTests` to disable particular pytest tests.
```

Comment:
```suggestion
 - `disabledTests` to disable particular pytest tests.
 - `disabledTestFiles` to disable particular pytest files.
```

**disabledTestFiles** is currently on staging and soon master.

Reply:
Would you prefer I remove the reference to `disabledTestFiles`?